panoramapublic/src/org/labkey/panoramapublic/query/CatalogEntryManager.java

```diff
@@ -144,9 +147,10 @@ public static void moveEntry(@NotNull ExperimentAnnotations previousCopy, @NotNu
         }
     }
 
-    public static List<CatalogEntry> getApprovedEntries(int entryCount)
+    public static List<CatalogEntry> getEntries(@NotNull CatalogEntryType entryType, int entryCount)
```

Comment:
Does entryCount just cap the number of rows it should return? If so, I think you could call `setMaxRows(entryCount)` on the `TableSelector`

Reply:
Yes, this could be simplified with `setMaxRows(entryCount)` and `setOffset(offset)`.  I will do that in the next iteration.